source/Handlebars/DynamicMetaObjectHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Dynamic;
+using System.Linq.Expressions;
+using System.Runtime.CompilerServices;
+using Microsoft.CSharp.RuntimeBinder;
+
+namespace Handlebars
+{
+    internal static class DynamicMetaObjectHelper
+    {
+        internal static object GetProperty(IDynamicMetaObjectProvider target, string name)
+        {
+            var self = (Expression)Expression.Constant(target);
```

Comment:
I think this is an unneeded cast?

Reply:
Yes, because ConstantExpression inherits from Expression. I will remove unneeded cast. Thanks.